sw/device/tests/crt_test.c

```diff
@@ -54,6 +55,99 @@ static void init_uart(void) {
   base_uart_stdout(&uart0);
 }
 
+static void test_crt_section_clear(void) {
```

Comment:
Could you please add a high level description here (using an array to simulate clearing sections, etc..).

Reply:
Done